apps/src/lib/node/ledger/events.rs

```diff
@@ -0,0 +1,101 @@
+use std::collections::HashMap;
+use std::fmt::{self, Display};
+use std::ops::{Index, IndexMut};
+
+use sha2::{Digest, Sha256};
+use tendermint_proto::abci::EventAttribute;
+
+/// Custom events that can be queried from Tendermint
+/// using a websocket client
+#[derive(Clone)]
+pub struct Event {
+    event_type: EventType,
+    attributes: HashMap<String, String>,
+}
+
+/// The two types of custom events we currently use
+#[derive(Clone)]
+pub enum EventType {
+    // The transaction was accepted to be included in a block
+    Accepted,
```

Comment:
This case looks unused right now. Is it for later?

Reply:
Yes, I was thinking we would want an event for wrapped transactions getting added onto the chain and a later event when they are decrypted and applied.